### PR TITLE
Minor grammatical fixes

### LINKED ISSUE #3897: 'Nother MOTD Typo
image

"effected" should be "affected"

All hail the grammar Nazi.

### PR DESCRIPTION
**Pull request by tmad40blue**
Changed all instances of "effected" (which is incorrect) to "affected".

Fixes #3897

### CODE REVIEW

File: LICENSE-AGPL3.txt [hunk 1]
```diff
@@ -174,7 +174,7 @@ measures.
 
   When you convey a covered work, you waive any legal power to forbid
 circumvention of technological measures to the extent such circumvention
-is effected by exercising rights under this License with respect to
+is affected by exercising rights under this License with respect to
```
Comment (tigercat2000) [inline LICENSE-AGPL3.txt]: Changing the licensing for grammar isn't okay. Effected is the correct term here, regardless.

File: code/controllers/ProcessScheduler/LICENSE-AGPL [hunk 2]
```diff
@@ -65,7 +65,7 @@ Conveying under any other circumstances is permitted solely under the conditions
...
 No covered work shall be deemed part of an effective technological measure under any applicable law fulfilling obligations under article 11 of the WIPO copyright treaty adopted on 20 December 1996, or similar laws prohibiting or restricting circumvention of such measures.
 
-When you convey a covered work, you waive any legal power to forbid circumvention of technological measures to the extent such circumvention is effected by exercising rights under this License with respect to the covered work, and you disclaim any intention to limit operation or modification of the work as a means of enforcing, against the work's users, your or third parties' legal rights to forbid circumvention of technological measures.
+When you convey a covered work, you waive any legal power to forbid circumvention of technological measures to the extent such circumvention is affected by exercising rights under this License with respect to the covered work, and you disclaim any intention to limit operation or modification of the work as a means of enforcing, against the work's users, your or third parties' legal rights to forbid circumvention of technological measures.
```
Comment (tigercat2000) [inline code/controllers/ProcessScheduler/LICENSE-AGPL]: Changing the licensing for grammar isn't okay. Effected is the correct term here, regardless.

**Comment by DaveTheHeadcrab** [comment]
Good lord those commits.
_All those commits_.

File: code/controllers/ProcessScheduler/LICENSE-AGPL (hunk 2 above)
Comment (tmad40blue) [inline code/controllers/ProcessScheduler/LICENSE-AGPL]: Regardless of whether changing the license for grammatical errors is "okay" - how is "effected" correct in this case?

**Comment by tigercat2000** [comment]
@tmad40blue this doesn't even fix that issue because the motd is a config option

**Comment by tmad40blue** [comment]
@tigercat2000 

That's the only instance of that MOTD message in the codebase - so where would you propose the correction be made?

**Comment by tigercat2000** [comment]
@tmad40blue It's a serverside config option. Only our staff can change it.

**Comment by tmad40blue** [comment]
@tigercat2000 

Sorry, I'm still confused. I corrected a grammatical error that was referenced in an earlier issue.

Where else would this correction be implemented if "only your staff" are allowed to change certain files?

Is this not the goal of accepting pull requests, to be able to correct issues and improve your codebase based on community contributions?

**Comment by Fox-McCloud** [comment]
13 commits for grammatical changes.

Not going to fly; just sayin', up front, right now.

File: code/controllers/ProcessScheduler/LICENSE-AGPL (hunk 2 above)
Comment (Aurorablade) [inline code/controllers/ProcessScheduler/LICENSE-AGPL]: http://grammarist.com/usage/affect-effect/

**Comment by tmad40blue** [comment]
@Fox-McCloud 

It's clear now that you do not wish to improve your codebase's grammar based on community contributions. I can understand that, but some forewarning would have been appreciated - perhaps a removal of your "Grammar and Formatting" tag, or some more detailed instruction as to how you would like future pull requests to be submitted, would assist you with this.

**Comment by DaveTheHeadcrab** [comment]
@tmad40blue 
What Tigercat is saying and what is, evidently, not getting through to you is that the MOTD is an option that is determined by _a serverside configuration option completely independent of the git._.

**Comment by Aurorablade** [comment]
@tmad40blue the issue at hand for the fix wasn't something that can be changed in the codebase..and was set by an admin. so in essence, the issue you are fixing..you are not fixin..and can only be fixedd by an admin.

**Comment by Aurorablade** [comment]
and if you use other PRs as examples..you can can do changes of multiple files in one commit...and changing anything in a legal licensing document without permission grammar or not..is a common sense no no.

**Comment by tmad40blue** [comment]
@Aurorablade 

I understand about the licenses at this point (although the grammar of "effected" is still incorrect); however, I'm still unsure why there is the IMMEDIATE hostility toward someone who is just trying to help out.

To you personally, thank you for clearly explaining what the actual issue is so that I can do it a little better next time. This explanation was what was lacking from previous replies.

**Pull request closed without merging**